src/parser.c

```diff
@@ -3213,19 +3211,22 @@ static void comment_insert_new_line(rbs_allocator_t *allocator, rbs_comment_t *c
 static rbs_comment_t *alloc_comment(rbs_allocator_t *allocator, rbs_token_t comment_token, rbs_comment_t *last_comment) {
     rbs_comment_t *new_comment = rbs_allocator_alloc(allocator, rbs_comment_t);
 
+    size_t initial_line_size = 10;
```

Comment:
This is the same initial size this code used to use before. I haven't spent any time tuning it.